packets.go

```diff
@@ -1063,19 +1064,26 @@ func (stmt *mysqlStmt) writeExecutePacket(args []driver.Value) error {
 					paramValues = append(paramValues, 0x00)
 				}
 
-			case []byte:
+			case []byte, json.RawMessage:
+				var (
+					ok bool
+					b  []byte
+				)
+				if b, ok = v.(json.RawMessage); !ok {
+					b = v.([]byte)
+				}
```

Comment:
I don't like this....  How about putting this code before `switch v := arg.(type) {`?

```go
if x, ok := arg.(RawMessage); ok {
    arg = arg.([]byte)
}
```